niworkflows/interfaces/bids.py

```diff
@@ -43,8 +52,7 @@ def _none():
 
 class _BIDSBaseInputSpec(BaseInterfaceInputSpec):
     bids_dir = traits.Either(
```

Comment:
I've left this for backwards compatibility, but this input is not necessary anymore.

Reply:
should the `deprecated` field be set then?

Reply:
If we want to signal ourselves that the input can be removed in future releases, okay. But I'm honestly not concerned about this particular one.